### PR TITLE
#434 - Added permission to overwrite tag

### LINKED ISSUE #434: Support overwrite permission for tags
Let's add dedicated permission for overwriting tags in the repository, so the user with write but without overwrite permission will be able to upload new layers, but won't be able to overwrite existing tags.

Permission name: `overwrite`.
The default behavior is: if user has write permission, then he can overwrite tags too.

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #434 
Added permission to overwrite tag. Please, note that user with `overwrite` permission can also create new tags.

### CODE REVIEW

**Review by g4s8 — DISMISSED** [review]
@olenagerasimova thanks

**Comment by g4s8** [comment]
@olenagerasimova please check tests are faiing

File: src/main/java/com/artipie/docker/http/Scope.java [hunk 1]
```diff
@@ -231,6 +231,23 @@ public String action() {
                 super(new Repository(name, "push"));
             }
         }
+
+        /**
+         * Scope for push action on repository resource.
+         *
+         * @since 0.10
+         */
```
Comment (genryxy) [inline src/main/java/com/artipie/docker/http/Scope.java:239]: @olenagerasimova this comment is irrelevant

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — DISMISSED** [review]
@olenagerasimova thanks, looks good to me

**Comment by genryxy** [comment]
@olenagerasimova try to update testcontainers version to `1.15.1` (https://github.com/testcontainers/testcontainers-java/issues/3574#issuecomment-742080370). This update helped in `npm-adapter`

**Comment by olenagerasimova** [comment]
@genryxy testcontainers are not used in docker-adapter((

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**